### PR TITLE
proxychains-ng: new package

### PR DESCRIPTION
**Pull request by dbermond**
**Compile tested:** R7800 (arm_cortex-a15_neon-vfpv4), OpenWrt git master
**Run tested:** R7800 (arm_cortex-a15_neon-vfpv4), OpenWrt git master

### CODE REVIEW

File: net/proxychains-ng/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+define Package/proxychains-ng/install
+	# binary executable
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/proxychains4 $(1)/usr/bin/
+	$(LN) proxychains4 $(1)/usr/bin/proxychains
+	
+	# library
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libproxychains4.so $(1)/usr/lib/
```
Comment (neheb) [inline net/proxychains-ng/Makefile]: This could probably be switched to INSTALL_DATA. Libraries don't need to be executable.

**Comment by dbermond** [comment]
Hi @neheb

Makefile updated following your review.

Many libraries are installed with filemode 755 by default.

**Comment by neheb** [comment]
Sure. There's no good reason for them to be though.

Anyway, I see no other problems. Will wait for others to chime in.

**Comment by dbermond** [comment]
@neheb Thanks for your review :)

**Comment by dbermond** [comment]
@BKPepe Makefile updated following your review. Spaces added.

Thank you for the review and approval.

**Comment by neheb** [comment]
Merged.

**Comment by dbermond** [comment]
@neheb Thanks for the merge

**Pull request merged**